web/src/elements/cards/AggregateCard.ts

```diff
@@ -8,8 +8,16 @@ import PFCard from "@patternfly/patternfly/components/Card/card.css";
 import PFFlex from "@patternfly/patternfly/layouts/Flex/flex.css";
 import PFBase from "@patternfly/patternfly/patternfly-base.css";
 
+export interface IAggregateCard {
+    icon?: string;
+    header?: string;
+    headerLink?: string;
+    subtext?: string;
+    leftJustified?: boolean;
```

Comment:
The Lit and Open-WC "best practices" guidelines state that boolean values should _never_ be true by default; since "isCenter" was the default behavior (see lines 26-34 immediately below) and was true by default, I've re-arranged the logic so that it's now, well, the default behavior, and "left-justified" in the component's attributes will disable that default behavior. 

Only one aggregate card ever uses the left-justified form anyway, and it's, yes, QuickActionsCard.